common/djangoapps/student/models.py

```diff
@@ -963,18 +963,25 @@ def is_course_full(self, course):
 
         return is_course_full
 
-    def users_enrolled_in(self, course_id, include_inactive=False):
+    def users_enrolled_in(self, course_id, include_inactive=False, exclude_fake_email=False):
         """
         Return a queryset of User for every user enrolled in the course.  If
         `include_inactive` is True, returns both active and inactive enrollees
-        for the course. Otherwise returns actively enrolled users only.
+        for the course. Otherwise returns actively enrolled users only. If
```

Comment:
Nitpick: if you drop `If` down and start the sentence on a new line, the diff ends up a little cleaner.